bin/tacl_cl_parser.py

```diff
@@ -243,51 +250,40 @@ def process_xml(xml: Path, is_tacl: bool) -> Optional[etree.Element]:
 
         paper.append(author)
 
-    year_text = get_year(front)
-    year = etree.Element("year")
-    year.text = year_text
-    paper.append(year)
-
-    month_text = get_month(front)
-    if month_text is not None:
-        month = etree.Element("month")
-        month.text = month_text
-        paper.append(month)
-
     doi_text = get_doi(front)
     doi = etree.Element("doi")
     doi.text = doi_text
     paper.append(doi)
 
     abstract_text = get_abstract(front)
-    abstract = etree.Element("abstract")
-    abstract.text = abstract_text
-    paper.append(abstract)
+    if abstract_text:
+        abstract = etree.Element("abstract")
+        abstract.text = abstract_text
+        paper.append(abstract)
 
```

Comment:
FYI there is also `make_simple_element` in `anthology.utils`, if you like:

```python
make_simple_element("abstract", abstract_text, parent=paper)
```

It returns the node in case you need to use it more, and also takes an `attrib` argument with a hash for attrib values.